en/components/select.md

```diff
@@ -88,7 +88,7 @@ The Select comes with the combined styling flexibility of its input and dropdown
 
 ## Usage
 
-When using the Select, its input and dropdown should have the same width and their left and right borders should match. When the Select is focused, the dropdown can be shown above the input and hide it until an item is chosen from the list, and you should always show the dropdown on top of other content that is on the same surface with the Select input. The Select doesn't push content as the expansion panel does.
+When using the Select, its input and drop down should have the same width and their left and right borders should match. When the Select is focused, the drop down is shown above or below the input, never over the input, since the user should always be informed about the current selection. Once an item from the list is selected the drop down gets hidden. The drop down on the other hand should be on top of other content that is on the same surface as the Select input. The Select doesn't push content as the expansion panel does.
```

Comment:
Once an item from the list is selected, the drop down gets hidden. (There's a comma after selected)